conans/util/files.py

```diff
@@ -373,3 +374,12 @@ def human_size(size_bytes):
         formatted_size = str(round(num, ndigits=the_precision))
 
     return "%s%s" % (formatted_size, the_suffix)
+
+
+# FIXME: completely remove disutils once we don't support <3.8 any more
```

Comment:
We can also make this feature of local-recipes-index to require Python >= 3.8 if that helps?